components/ImageInput.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+
+import { Button } from "lbh-frontend-react/components/Button/Button";
+
+export interface ImageInputProps {
+  images: string[];
+  label: string;
+  name: string;
+  buttonText: string;
+  onValueChange: (value: string) => void;
+}
+
+const toBase64 = (file: any) =>
+  new Promise((resolve, reject) => {
+    const reader = new FileReader();
+    reader.readAsDataURL(file);
+    reader.onload = () => resolve(reader.result);
+    reader.onerror = error => reject(error);
+  });
+
+export const ImageInput = (props: ImageInputProps): JSX.Element => {
+  const { images, label, buttonText, name } = props;
+
+  const [thumbnails, setThumbnails] = useState(images);
+
+  return (
+    <div className="imageComponent">
+      {thumbnails && (
+        <div className="thumbnails">
+          {thumbnails.map(thumbnail => (
+            <div className="thumbnail">
+              <img className="uploadedImage" src={thumbnail} />
+              <Button
+                className="govuk-button--warning lbh-button--warning image-input-button--remove"
+                onClick={() =>
+                  setThumbnails(
+                    thumbnails.filter(thumbnail => thumbnail !== thumbnail)
+                  )
+                }
+              >
+                Remove photo
+              </Button>
+            </div>
+          ))}
+        </div>
+      )}
+
+      <label>
+        {label}
+        <input
+          id={name}
+          name={name}
+          type="file"
+          accept="image/*"
+          capture
+          onChange={async event => {
+            const base64File = await toBase64(event.target.files![0]);
+            setThumbnails([...thumbnails, base64File as string]);
```

Comment:
Watch out for updating state asynchronously. If you update the state after a component has unmounted, it will throw an exception. This needs to only call `setThumbnails` if it's mounted.